test-cases/python/critical-business-logic.py

```
@@ -0,0 +1,130 @@
+# Test case: Business logic implementation
+
+import datetime
+import decimal
```

Ruff (0.12.2)

`decimal` imported but unused

Remove unused import: `decimal`

(F401)